llama-index-integrations/embeddings/llama-index-embeddings-cohere/llama_index/embeddings/cohere/base.py

```diff
@@ -74,14 +74,16 @@ class CohereEmbedding(BaseEmbedding):
     # Instance variables initialized via Pydantic's mechanism
     cohere_client: Any = Field(description="CohereAI client")
     truncate: str = Field(description="Truncation type - START/ END/ NONE")
-    # input_type: Optional[str] = Field(description="Model Input type")
+    input_type: Optional[str] = Field(
```

Comment:
what would be the plan to deprecate the `input_type` param in future releases? It's totally fine to keep it now to make this a non-breaking change but I worry that this could get confusing longterm

Reply:
I don't think its confusing? Especially since cohere has more than two modes, you might want to override. For example, clustering